packages/core/utils/create-project.ts

```diff
@@ -1,28 +1,11 @@
 import * as path from "path";
-import { booleanPrompt, selectPrompt, textPrompt } from "./prompts.js";
+import { PromptOption, booleanPrompt, selectPrompt, textPrompt } from "./prompts.js";
 import { commonFilePaths, directoryExists, fileExists } from "../files/utils.js";
```

Comment:
we'll want to denote `PromptOption` as a type and add `endPrompts` (see below)
```suggestion
import { type PromptOption, booleanPrompt, selectPrompt, textPrompt, endPrompts } from "./prompts.js";
```